### PR TITLE
backport: trivial 2023 12 27

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## What was done?
Trivial backports

## How Has This Been Tested?
Building; make check

## Checklist:
  _Go over all the following points, and put an `x` in all the boxes that apply._
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: doc/build-netbsd.md [hunk 1]
```diff
@@ -60,7 +60,6 @@ With wallet:
 ./configure --with-gui=no CPPFLAGS="-I/usr/pkg/include" \
     LDFLAGS="-L/usr/pkg/lib" \
     BOOST_CPPFLAGS="-I/usr/pkg/include" \
-    BOOST_LDFLAGS="-L/usr/pkg/lib" \
```
Comment (knst) [inline doc/build-netbsd.md:63]: bitcoin#24524 can be done only after bitcoin#24301

Comment (PastaPastaPasta) [reply doc/build-netbsd.md]: dropped

**Review by knst — APPROVED** [review]
utACK (restarted CI for tsan: please validate succeed before merge)

**Comment by PastaPastaPasta** [comment]
(CI succeeded; rebased on latest develop, merging so I can do another PR @UdjinM6 please post merge review)

**Review by UdjinM6 — COMMENTED** [review]
post-merge-utACK

**Pull request merged**